test/dynamo/test_fake_distributed.py

```diff
@@ -30,6 +34,7 @@ def setUp(self):
 
     def tearDown(self):
         dist.destroy_process_group()
+        dist.distributed_c10d._world = None
```

Comment:
:0 how come. I have lots of code that's only doing the first line, do I have to do this one too?